JavaDirectory/.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,5 @@
+eclipse.preferences.version=1
+encoding//src/main/java=UTF-8
+encoding//src/main/resources=UTF-8
+encoding//src/test/java=UTF-8
```

Comment:
.settings/ folder does not need to be committed

Reply:
done